.github/workflows/e2e.sign-attestations.schedule.yml

```
@@ -44,13 +44,13 @@ jobs:
         with:
           node-version: 16
       - name: install sigstore-js
-        run: npm ci
+        run: npm install -g @sigstore/cli@^0.8.0
       # TODO: Add subject verification. This only verifies the signature on the DSSE envelope.
       - name: run verify
         run: |
           set -euo pipefail
           for file in outputs/*; do
-            ./node_modules/.bin/sigstore verify "$file"
+            sigstore verify "$file"
```

I think using `npm ci` above and then just running via npx would be better.
```suggestion
            npx sigstore verify "$file"
```

This is essentially what I was doing before but running via `npx` is probably nicer than referencing the executable directly in `node_modules/.bin`.

---

Thanks, that's a better place for this.